pkg/client/smd.go

```diff
@@ -1,22 +1,16 @@
-package configurator
+package client
 
 import (
 	"bytes"
-	"crypto/tls"
-	"crypto/x509"
 	"encoding/json"
 	"fmt"
 	"io"
-	"net"
 	"net/http"
-	"os"
-	"time"
 
-	"github.com/OpenCHAMI/configurator/pkg/util"
+	configurator "github.com/OpenCHAMI/configurator/pkg"
+	"github.com/caarlos0/log"
```

Comment:
Any reason you're using a different logging package from zerolog here?

When attempting to build with this, I get:
```
# github.com/OpenCHAMI/configurator/pkg/client
pkg/client/smd.go:61:3: not enough arguments in call to log.Info
	have ()
	want (string)
pkg/client/smd.go:61:3: log.Info() (no value) used as value or type
pkg/client/smd.go:103:3: not enough arguments in call to log.Info
	have ()
	want (string)
pkg/client/smd.go:103:3: log.Info() (no value) used as value or type
pkg/client/smd.go:142:3: not enough arguments in call to log.Info
	have ()
	want (string)
pkg/client/smd.go:142:3: log.Info() (no value) used as value or type
```

Reply:
That's just `goimports` being stupid. Let me change it real quick.